electron/main.ts

```diff
@@ -1,36 +1,38 @@
-import * as path from 'path'
-
+import * as path from 'path';
 import {
   BrowserWindow,
   Menu,
   Notification,
   Tray,
   app,
+  dialog,
   ipcMain,
   powerSaveBlocker,
   protocol
-} from 'electron'
-import { cpus } from 'os'
+} from 'electron';
+import {
+  cpus,
+  platform
+} from 'os';
 import {
   exec,
   spawn
-} from 'child_process'
+} from 'child_process';
```

Comment:
No semicolons please.

Reply:
On Windows my settings are different than on Linux so its putting this when auto-import. Will fix.